metainfo/org.devsk.flatconvert.metainfo.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component
+    xmlns="http://www.freedesktop.org/standards/appstream"
+    xmlns:its="http://www.w3.org/2005/11/its"
+    type="desktop">
+  <id>org.devsk.flatconvert.desktop</id>
+  <name its:translate="yes">flatconvert</name>
+  <summary its:translate="yes">A file conversion application for the gnome desktop environnement.</summary>
+  <url type="homepage">https://gitlab.gnome.org/qsk/flatconvert</url>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>GPL-3.0-or-later</project_license>
+  <description its:translate="yes">
+    <p>A file conversion application for the gnome desktop environnement.</p>
+  </description>
+</component>
```

Comment:
missing required `url`, `content_rating`, `releases`, `screenshots`.

Also `description` should not be a repeat of `summary`.